src/com/intellij/idea/plugin/hybris/project/configurators/impl/DefaultLibRootsConfigurator.kt

```diff
@@ -71,7 +71,7 @@ class DefaultLibRootsConfigurator : LibRootsConfigurator {
                 if (moduleDescriptor.hasBackofficeModule) {
                     val backofficeJarDirectory = File(moduleDescriptor.moduleRootDirectory, HybrisConstants.BACKOFFICE_JAR_PATH)
                     if (backofficeJarDirectory.exists()) {
-                        YModuleLibDescriptorUtil.addRootProjectLibrary(modifiableModelsProvider, backofficeJarDirectory, HybrisConstants.BACKOFFICE_LIBRARY_GROUP)
+                        YModuleLibDescriptorUtil.addRootProjectLibrary(modifiableModelsProvider, backofficeJarDirectory, null, HybrisConstants.BACKOFFICE_LIBRARY_GROUP)
```

Comment:
instead of passing null we can use named parameters for method:
```
YModuleLibDescriptorUtil.addRootProjectLibrary(
modifiableModelsProvider = modifiableModelsProvider, 
libraryDirRoot = backofficeJarDirectory, 
libraryName = HybrisConstants.BACKOFFICE_LIBRARY_GROUP
)
```